cmd/fossa/config.go

```diff
@@ -21,10 +21,11 @@ type configFileV1 struct {
 
 	CLI struct {
 		// Upload configuration.
-		APIKey  string `yaml:"api_key"`
-		Server  string
-		Project string
-		Locator string
+		APIKey   string `yaml:"api_key"`
+		Server   string
+		Fetcher  string // for now we don't allow users to set this
+		Project  string
+		Revision string
```

Comment:
A few notes here:

- I'm not sure how I feel about changing this because we already have customers using `fossa` live and this is potentially a breaking change to their configurations.
- That said, I don't think anybody sets the `Locator` field, so changing this should be relatively safe.
- Users can already specify `project` and `revision` via command line flags. This struct only defines the format of the configuration file, and is different from the shape of the configuration struct we use internally. I think I might actually prefer taking out `Locator` and not putting anything else in -- when would anybody ever want to specify revision via configuration file?